src/yin_yang.py

```diff
@@ -73,7 +74,7 @@ def run(self):
                 except Exception as e:
                     logger.error('Error while changing the theme in plugin ' + pl.name)
                     traceback.print_exception(type(e), e, e.__traceback__)
-        play_sound("./assets/dark.wav")
```

Comment:
I would much rather like to see the option to play the sound or not. You can set the path to the dark.wav to a empty path to disable this.